README.md

```diff
@@ -24,9 +24,47 @@ Or install it yourself as:
 
     $ gem install searchmetrics_client
 
+## Configuration
+
+To start using gem you have to configure API keys as bellow:
+
+```ruby
+SearchmetricsClient.configure do |config|
+  config.api_key    = 'Your api key'
+  config.api_secret = 'Your api secret'
+end
+```
+
+Additionally you can specify `api_version` and `api_url`, which defaults to `v1` and `http://api.searchmetrics.com` regardless.
+
 ## Usage
 
-TODO: Write usage instructions here
+To get metric from Searchmetrics server use:
+
+```ruby
+SearchmetricsClient::Request.send_request_form_hash(product: :research,
+                                                    data_category: :keywords,
+                                                    method: :get,
+                                                    return_data: :list,
+                                                    type: :keywordinfo,
+                                                    params: { countrycode: 'PL',
+                                                              keyword: 'ruby' }
+                                                   )
+```
+
+Key `type` is required. The rest of values will have assigned default values if not specified:
+Defaults are:
+
+```ruby
+product: :research
+data_category: :links
+method: :get
+return_data: :list
+params: { }
+```
+
+TODO: Write more usage instructions here
```

Comment:
Could you remove the `TODO` and add how the data structure looks like which is returned by the call you are showing?